en/option/series/gauge.md

```diff
@@ -126,6 +126,16 @@ Default value:
 [[1, '#E6EBF8']]
 ```
 
+Note: The value of `color[i][0]` means the percentage of the axis line of gauge chart's segments.
+
+```js
+color: [
+    [0.1, 'red'], // 10% red segments
+    [2, 'green'], // rest percent green segments
```

Comment:
We should not encourage users to set percentage larger than 1. The blue color in your case doesn't work not because 10 is larger than 1, but because its previous percentage, 2, is larger than 1. So perhaps it is better to tell users the percentage can be between 0 and 1. That should be enough.

Reply:
Ok, I will change the case in the next commit, which makes the value must be between 0 and 1. Just remind users must not do that case in their usage from it, bcs that does not work.